endoreg_db/models/media/video/pipe_1.py

```
@@ -61,7 +64,16 @@ def _pipe_1(
                     model_meta = ai_model_obj.get_latest_version()
             except AiModel.DoesNotExist:
                 logger.error(f"Pipe 1 failed: Model '{model_name}' not found.")
```

Ruff (0.14.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)